saas/fsm/renewal_reminder/README.md

```diff
@@ -0,0 +1,29 @@
+# SaaS Manager - Renewal Reminder
+
+## What it does
+
+This policy will raise an alert when Flexera SaaS Manager identifies applications whose contract expiration date is approaching.
```

Comment:
product doesn't want to use alert. instead change **raise an alert** to **create an incident.**